SPECS/ruby/ruby.spec

```diff
@@ -50,20 +51,20 @@ Provides:       rubygem(io-console) = %{version}-%{release}
 Provides:       rubygem(psych) = %{version}-%{release}
 Provides:       rubygem(did_you_mean) = %{version}-%{release}
 Provides:       rubygem(irb) = %{version}-%{release}
+Provides:       rubygem(json) = %{version}-%{release}
```

Comment:
I see that's the way it's done for the other gems, but do you know why we use the spec's version while `json_version` is completely different?

Reply:
This is an artifact of how we packaged these provides for 1.0. I'm following this pattern to avoid breaking any downstream specs.

But yes, we definitely should replace these with the actual gem versions. Ideally this would happen at the same time as a migration to Ruby 3.X, at which time we would also extend the scope of which standard library modules are installed into our gem directory. 

This sort of change would probably require either an Epoch tag, or we would need approval to make a breaking change that could affect derivative packagers. Given how we've proceeded with 2.0 changes so far... probably the latter.

I'll go ahead and add a comment in the spec saying all of this.